doc/modules/model_evaluation.rst

```diff
@@ -163,6 +159,24 @@ the :func:`fbeta_score` function::
     >>> grid = GridSearchCV(LinearSVC(dual="auto"), param_grid={'C': [1, 10]},
     ...                     scoring=ftwo_scorer, cv=5)
 
+==============================================     ==================================
+Function                                           Comment
+==============================================     ==================================
+**Classification**
+:func:`metrics.fbeta_score`                        implements parameter ``beta``
+:func:`metrics.hamming_loss`
+:func:`metrics.zero_one_loss`
```

Comment:
```suggestion
```

I think we should remove those because they do not have user settable parameters and focus this PR on metrics which require user-settable parameters.